src/ngx_http_encrypted_session_cipher.c

```diff
@@ -26,20 +26,20 @@ ngx_http_encrypted_session_aes_mac_encrypt(ngx_pool_t *pool, ngx_log_t *log,
     const u_char *in, size_t in_len, ngx_uint_t expires, u_char **dst,
     size_t *dst_len)
 {
-    EVP_CIPHER_CTX           ctx;
     const EVP_CIPHER        *cipher;
     u_char                  *p, *data;
     int                      ret;
     size_t                   block_size, buf_size, data_size;
     int                      len;
     uint64_t                 expires_time;
     time_t                   now;
+    EVP_CIPHER_CTX          *ctx = EVP_CIPHER_CTX_new();
```

Comment:
We should avoid using non-constant expressions in C variable declaration statements. This violates the nginx coding disciplines. Also, we need to check if `EVP_CIPHER_CTX_new()` returns `NULL`.